xss_utils/django/README.rst

```diff
@@ -0,0 +1 @@
+- Contains XSS utilities specific to Django Templates.
```

Comment:
Let's drop the django directory for now and move this README to templatetags, and adjust it to read:
```
- Contains custom tags for Django Templates to prevent XSS.